content/docs/command-reference/spec.json

```diff
@@ -0,0 +1,734 @@
+{
+  "apply": {
+    "args": {
+      "args": [
+        {
+          "decl": "model",
+          "help": "Path to model object [required]"
+        },
+        {
+          "decl": "data",
+          "help": "Path to data object [required]"
+        }
+      ],
+      "impls": null,
+      "impl_metavar": null,
+      "subcommands": null
+    },
+    "options": [
+      {
+        "decl": "-p, --project TEXT",
+        "help": "Path to MLEM project [default: (none)]"
+      },
+      {
+        "decl": "--rev TEXT",
+        "help": "Repo revision to use [default: (none)]"
+      },
+      {
+        "decl": "-o, --output TEXT",
+        "help": "Where to store the outputs."
+      },
+      {
+        "decl": "-m, --method TEXT",
+        "help": "Which model method is to be applied [default: predict]"
+      },
+      {
+        "decl": "--data-project, --dr TEXT",
+        "help": "Project with data"
+      },
+      {
+        "decl": "--data-rev TEXT",
+        "help": "Revision of data"
+      },
+      {
+        "decl": "-i, --import",
+        "help": "Try to import data on-the-fly"
+      },
+      {
+        "decl": "--import-type, --it TEXT",
+        "help": "Specify how to read data file for import. Available types: ['pandas', 'pickle']"
```

Comment:
This one doesn't get a `\n` but then it goes over 80 characters [when rendered](https://github.com/iterative/mlem.ai/pull/172/files#diff-25ae2c278dc18c37bad2aefdfe95eabd2602853ae521150491b95bbf62e33143R39)...

Reply:
fixed

Reply:
Thanks